### ISSUE TITLE
Licenses do not appear in .plist if repo URL ends in a slash

### ISSUE THREAD

**Issue by gereons** [mono0926/LicensePlist#159]
I have an app that uses https://github.com/marmelroy/PhoneNumberKit via SPM.

LicensePlist's output does not include the license for this libraries, while all others I'm using do show up, and I can't immediately see a difference from that library's setup from any others.

In the generated `latest_results.txt` File, I have this snippet that looks suspicious to me:

```
name: Zip, nameSpecified: 
body: The MIT License (MIT…
version: 2.1.1

name: , nameSpecified: PhoneNumberKit, owner: PhoneNumberKit, version: 3.3.3

name: abseil-cpp-SwiftPM, nameSpecified: abseil, owner: firebase, version: 0.20200225.4
```

(the entry for PhoneNumberKit is the only one that does not have a name)

**Comment by gereons** [comment]
I just had the same effect on another package when I switched from the main repo (https://github.com/sindresorhus/Defaults) to a branch in a fork (https://github.com/automactic/Defaults/tree/xcode13).

Is this issue maybe related to the fact that the branch isn't named `main`?

**Comment by gereons** [comment]
I did some further tests and found that this behaviour depends on whether or not the repository URL ends in a slash or not:

`https://github.com/sindresorhus/Defaults`  works, but 
`https://github.com/sindresorhus/Defaults/` doesn't.

This is of course easy to work around on my side, but you might still want to fix this as people are likely to copy and paste URLs that have this trailing slash.

**Comment by acevif** [comment]
I could reproduce this issue in commit 6d0b75e.

**Comment by acevif** [comment]
@mono0926 Are you working on this issue already?
Or Did you assign someone to this issue?

If not, I would appreciate it if you assign this issue to me.

**Comment by mono0926** [comment]
@acevif 

Not yet.

Thank you, I assigned you 🙏

**Comment by acevif** [comment]
@gereons, I think we have fixed this problem in [3.23.4](https://github.com/mono0926/LicensePlist/releases/tag/3.23.4).

If you find this problem still occurs, please feel free to reopen this issue.
